symengine/lib/symengine.pxd

```diff
@@ -671,6 +671,14 @@ cdef extern from "<symengine/matrix.h>" namespace "SymEngine":
             const DenseMatrix &x, DenseMatrix &result) nogil except +
     void diff "SymEngine::sdiff"(const DenseMatrix &A,
             RCP[const Basic] &x, DenseMatrix &result) nogil except +
+    void row_join(const DenseMatrix &A, const DenseMatrix &B, DenseMatrix &C) nogil
```

Comment:
But `add_matrix` by default has only two parameters. Changing the declarations similarly has no effect as well.